src/pipx/venv.py

```diff
@@ -212,6 +215,25 @@ def install_package(
         if self.package_metadata[package].package_version is None:
             raise PackageInstallFailureError
 
+    def install_package_no_deps(
+        self, package_or_url: str, pip_args: List[str]
+    ) -> Optional[str]:
+        package: Optional[str]
+
+        with animate(f"resolving {package_or_url!r}", self.do_animation):
```

Comment:
`resolving` seems ambiguous to me. What do you think of something more descriptive like

* `determining package name from {...}`
* `resolving package name from {...}`
* `getting package name from {...}`

Reply:
Yes, and "resolving" might be a little more loaded, it's probably too associated with package resolution.

Maybe determining package name?

Reply:
Done.  Changed to "determining package name from "